### ISSUE TITLE
DO_Digicam_control does not work with 3.5 (no way to take pictures)

### ISSUE THREAD

**Issue by Gun-jam** [diydrones/MissionPlanner#1135]
Hello

With pixhawk running 3.5 firmware the DO-DIGICAM_CONTROL does not function. This is my primary method of image collection.

The issue appears to be with Shutter CDM in the DO_DIGICAM_CONTROL options it is by default 0
Apparently now it must be a non zero value.

Setting this value to 1 causes the UAV to capture a photo it also causes the waypoint line to head way off the map im assuming to GPS 0,0 once the flight plan is uploaded...before upload it looks okay in the flight plan window....in the flight data window its no good.

The UAV still executes the flight plan as it should but now there is no reference because Mission planner  show flight path like off cost of Africa or something. 

I usually shoot 280+ images per flight...I can not set Shutter CDM value to 1 for each trigger location
Having MP show flight paths heading to Africa is a bit of a danger because now I cant tell if its a bug or the UAV is really going to Africa. 

Im hoping you can come up with some type of solution for this.
Thanks
-Nick
![img_0257](https://cloud.githubusercontent.com/assets/17127794/13024764/22a22968-d1af-11e5-96dd-c207437dab2f.jpg)
![img_0256](https://cloud.githubusercontent.com/assets/17127794/13024770/35c2d448-d1af-11e5-9453-bb4bfed7f071.jpg)

**Comment by Gun-jam** [comment]
further note... this occurs when using "Auto WP" to draw "survey (grid)" inside a polygon. and using DO digicam control as the method of trigger 

if you just point to a few places and add "do digicam control" using add below...it works fine.

**Comment by meee1** [comment]
thanks for the bug report, and the details you provided.

**Comment by Gun-jam** [comment]
any idea when this would be available to try out?

Thanks

**Comment by meee1** [comment]
now, should work.

**Comment by Gun-jam** [comment]
Okay so first of all looking back at issue 1129 (the distance trigger method) this does indeed work exactly as I need it to. The problem is I had to uninstall mission planner then reinstall then download the beta.

I did that exact same step now and everything seemed to install just fine but when I create a mission using survey (grid) and look at "shutter cmd" the value is still zero... I did not fly to test if it actually works yet but my feeling is NO...As far as I can tell if shutter cmd is 0 it wont take a picture.

Should I expect to see a 1 value for shutter cmd or did you solve the issue using a different method?

Thank you for your help!
